test/functional/feature_taproot.py

```diff
@@ -1076,6 +1078,13 @@ def big_spend_inputs(ctx):
 
     # == Test for sigops ratio limit ==
 
+    # BIP348 CSFS signatures are embedded directly into the tapleaves vs the witness stack
```

Comment:
This comment was a bit confusing to me - perhaps someting like: "BIP348 CSFS messages and signatures can be embedded directly into the tapleaves to simplify testing since they do necessarily cover the spending transaction"